EIPS/eip-7441.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 7441
+title: Upgrade block proposer election to Whisk
+description: Allow elected block proposers to remain private until block publishing, to prevent DoS attacks
+author: George Kadianakis (@asn-d6), Justin Drake (@JustinDrake), dapplion (@dapplion)
+discussions-to: https://ethereum-magicians.org/t/eip-7441-upgrade-block-proposer-election-to-whisk-ssle/15316
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-09-01
+---
+
+## Abstract
+
+Upgrades the block proposer election mechanism to Whisk, a single secret leader election (SSLE) protocol. Currently, block proposers are publicly known in advance, sufficiently to allow sequential DoS attacks that could disable Ethereum. This upgrade makes the next block proposer unknown until its block is published.
```

Comment:
```suggestion
Upgrades the block proposer election mechanism to Whisk - a single secret leader election (SSLE) protocol. Currently block proposers are publicly known in advance sufficiently to allow sequential DoS attacks that could halt the Ethereum blockchain. This upgrade makes the next block proposer unknown until the current block is published.
```

Reply:
yea it was correct. `compromise Ethereum's liveness` sounds great. updated the recommend. also would it be correct to replace `its block` with `the block` then? `its` sounds a bit confusing as to what it refers to

Reply:
So the meaning is that a proposer is only revealed when specifically the block it is proposing is revealed, and not any other. The word "its" implied that relationship between what block carries the reveal, just "the block" does not specify which block.

Reply:
aha this makes sense. could we change it to `This upgrade allows the next block proposer to stay secret and hidden until its block is published.` 

else current could also work but will provide less context to someone starting from zero background.